src/preact/comments.js

```diff
@@ -20,27 +20,21 @@
 import {h, Component} from 'preact';
 import VoteButtons from './vote-buttons';
 
-export default class Comments extends Component {
-  render () {
-    const comments = this.props.comments;
-    return (
-      <section class="post__comments">
-        <h1>Comments</h1>
-        {
-          comments.map((c, index) => {
-            return (
-              <div class="post__comment" key={index}>
-                <h2 class="post__comment-author">{c.username} wrote</h2>
-                <p class="post__comment-text">
-                  {c.text}
-                </p>
+const Comments = ({ comments }) =>
+  <section class="post__comments">
+    <h1>Comments</h1>
+    {
+      comments.map((c, index) =>
+        <div class="post__comment" key={index}>
+          <h2 class="post__comment-author">{c.username} wrote</h2>
+          <p class="post__comment-text">
+            {c.text}
+          </p>
 
```

Comment:
Comments is a stateless functional component, so I’ve switched to the arrow function pattern; the component is default-exported at the end of the file (b/c `export default const` isn’t valid).

I’ve also created `comments` via destructuring, so there’s no need for a `const` variable anymore - this is, I think, a Preact-specific feature.

Lastly, I’ve dropped all the `render`s and `return`s, which are not needed since the arrow functions implicitly return the JSX code.